.github/workflows/integration.yaml

```diff
@@ -9,31 +9,31 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        k8s_version: [v1.25.3, v1.24.7, v1.23.13]
+        k8s_version: [v1.31.1, v1.29.8, v1.28.13]
```

Comment:
It should include 1.31.x, 1.30.x, 1.29.x

Reply:
fixed.